lib/babelish/android2csv.rb

```diff
@@ -6,22 +6,40 @@ def initialize(args = {:filenames => []})
       super(args)
     end
 
+    def parse_comment_line(line)
+      line.strip!
+      if line[0] != ?# && line[0] != ?=
+        m = line.match(/^\/\*(.*)\*\/\s*$/)
```

Comment:
@netbe This function I copied from strings2csv.rb, so unsure if I should still change this according to houndci-bot.